barretenberg/cpp/src/barretenberg/stdlib/primitives/bigfield/bigfield.fuzzer.hpp

```diff
@@ -1930,6 +1992,7 @@ extern "C" int LLVMFuzzerInitialize(int* argc, char*** argv)
  */
 extern "C" size_t LLVMFuzzerCustomMutator(uint8_t* Data, size_t Size, size_t MaxSize, unsigned int Seed)
 {
+    // TODO(alex): not sure if it really matters right here
```

Comment:
What does this mean?

Reply:
The call to fuzzing helper is done using StandardCircuitBuilder, but all the operations that it's performing are not builder specific. Probably need to delete this comment